src/action.js

```diff
@@ -15,7 +15,8 @@ var geo_action = {
   // annotation actions
   annotation_line: 'geo_annotation_line',
   annotation_polygon: 'geo_annotation_polygon',
-  annotation_rectangle: 'geo_annotation_rectangle'
+  annotation_rectangle: 'geo_annotation_rectangle',
+  annotation_edit_handle: 'geo_annotation_edit_handle'
```

Comment:
should we just call it annotation_edit? or would that be confusing?

Reply:
We may eventually want other edit actions.  For instance, suppose we supported an action on selecting the annotation in edit mode, then it might be `edit_select` to differentiate it.